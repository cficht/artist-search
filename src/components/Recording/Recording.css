.Recording {
  display: flex;
  justify-content: center;  
  align-items: center;
  flex-direction: column;

  background-color: red;
  color: white;

  text-align: center;

  p {
    width: 50%;
    padding: 20px;
    border-radius: 10px;
    
    background-color: darkblue;
  }
}