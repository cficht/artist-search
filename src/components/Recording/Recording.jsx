import React from 'react';
import PropTypes from 'prop-types';
import styles from './Recording.css';

const Recording = ({ artist, recording, lyrics }) => (
  <section className={styles.Recording}>
    <h1>{`${artist} - ${recording}`}</h1>
    <pre>
      <p>{lyrics}</p>
    </pre>
  </section>
);

Recording.propTypes = {
  artist: PropTypes.string.isRequired,
  recording: PropTypes.string.isRequired,
  lyrics: PropTypes.string
};

export default Recording;
