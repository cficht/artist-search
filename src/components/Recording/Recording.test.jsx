import React from 'react';
import { shallow } from 'enzyme';
import Recording from './Recording';

describe('Recording component', () => {
  it('renders Recording', () => {
    const wrapper = shallow(<Recording 
      artist="The Beatles"
      recording="Please Please Me"
      lyrics="Oh yeah" />);
    expect(wrapper).toMatchSnapshot();
  });
});
