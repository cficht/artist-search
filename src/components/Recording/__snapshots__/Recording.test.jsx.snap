// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recording component renders Recording 1`] = `
<Fragment>
  <h1>
    The Beatles
  </h1>
  <h2>
    Please Please Me
  </h2>
  <p>
    Oh yeah
  </p>
</Fragment>
`;
